suite/suite_test.go

```diff
@@ -429,7 +429,7 @@ func (sc *StdoutCapture) StopCapture() (string, error) {
 	}
 	os.Stdout.Close()
 	os.Stdout = sc.oldStdout
-	bytes, err := ioutil.ReadAll(sc.readPipe)
+	bytes, err := io.ReadAll(sc.readPipe)
```

Comment:
This change requires Go 1.16. `testify` is used in a wide range of project. This is not a good thing to force users to upgrade Go while it is not necessary (`io/ioutil` will stay forever).